docs/faststore/docs/customization/building-sections/list-of-native-sections.mdx

```diff
@@ -9,7 +9,33 @@ Sections in FastStore are components that organize and encapsulate other compone
 
 For example, the [Hero](https://developers.vtex.com/docs/guides/faststore/organisms-hero) is a native section designed to capture your brand's identity and messaging. To effective function in the store, the Hero comprises three components: `Hero`, `HeroImage`, and `HeroHeader`.
 
----
+## Considerations while customizing default components
+
+Customizing `__experimentalProductCard` and `SearchInput` can impact your [Intelligent Search Analytics](https://help.vtex.com/en/tutorial/analytics--6qRMAHDL9hvv3oE0bh8mA1) store report. These components collect data for your store analytics. Prioritize maintaining tracking functionality.
```

Comment:
I think it's also good to add something like this in the begin:

Override some default core components can affect the starter store default behaviors, such as customizing ....